defmt/src/lib.rs

```diff
@@ -367,6 +367,42 @@ pub use defmt_macros::timestamp;
 /// ```
 pub use defmt_macros::bitflags;
 
+/// Users of the defmt crate can enable the bitflagsv2 feature by adding defmt = { features = ["bitflagsv2"] } 
```

Comment:
Nit: On the overview page, Rustdocs will list the first paragraph as the high-level description.

This paragraph is a little bit long for such a description, please add a short description like there is for the v1 macro
e.g.:
```
/// Generates a bitflags structure that can be formatted with defmt (using version 2 of the bitflags crate)
///
```